cvm-datadisk-enc-scripts/CVM-enable-conftempdiskenc-Win.ps1

```diff
@@ -50,7 +50,7 @@ if ($keyvault -eq $null)
 # ADE settings:
 $Publisher                   = "Microsoft.Azure.Security"
 $ExtName                     = "AzureDiskEncryption"
-$ExtHandlerVer               = "2.4"
+$ExtHandlerVer               = "2.5"
 $EncryptionOperation         = "EnableEncryption"
 
 # Settings for enabling temp disk encryption only providing Azure Key Vault resource.
```

Comment:
Let's add a comment that they key names are case-sensitive. (That's across all files)

Reply:
comment is added.